Covinator/Covinator/Pages/modernaVaccination.cshtml

```diff
@@ -1,34 +1,41 @@
 ﻿@page
 @model Covinator.Pages.modernaVaccinationModel
 @{
-    var modernaVaccineDistributionAllocationss = (QuickType.ModernaVaccineDistributionAllocations[])ViewData["ModernaVaccineDistributionAllocations"];
+    ViewData["Title"] = "Moderna Vaccination";
+    var modernaAllocations = (QuickType.ModernaVaccineDistributionAllocations[])ViewData["ModernaVaccineDistributionAllocations"];
 }
 
 <div class="text-center">
     <h1 class="display-4">Moderna Vaccine Distribution Allocations</h1>
     <hr />
-    <table border="1" weidth="90%">
-        <tr>
-            <th>Jurisdiction</th>
-            <th>Week of allocations</th>
-            <th>1st dose allocations</th>
-            <th>2nd dose allocations</th>
-            @{ foreach (QuickType.ModernaVaccineDistributionAllocations modernaVaccineDistributionAllocations in modernaVaccineDistributionAllocationss)
-                {
-                <tr>
-                    <td>@modernaVaccineDistributionAllocations.Jurisdiction </td>
-                    <td>@modernaVaccineDistributionAllocations.WeekOfAllocations of allocations </td>
-                    <td>@modernaVaccineDistributionAllocations.The1StDoseAllocations </td>
-                    <td>@modernaVaccineDistributionAllocations.The2NdDoseAllocations </td>
+  
+       @if (modernaAllocations.Length > 0)
```

Comment:
check length of array to avoid displaying empty table header
Source:-
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions
https://docs.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-5.0&tabs=visual-studio